### ISSUE TITLE
RCT1 go_kart and corkscrew_train objects missing from rct1 ride objects directory (LINUX)

### ISSUE THREAD

**Issue by MrCantaloupe** [OpenRCT2/OpenRCT2#19432]
### Operating System

Linux (Ubuntu 22.04.1 LTS)

### OpenRCT2 build

OpenRCT2, v0.4.3 (, DEBUG)   [probably gf5265109c]

### Base game

RollerCoaster Tycoon 2

### Area(s) with this issue?

This issue has to do with park objects and/or asset packs

### Describe the issue

Parks containing RCT1 go karts and corkscrew trains fail to load with the error message "The file could not be loaded as some of the objects referenced in it are missing or corrupt", and will list rct1.ride go_karts or corkscrew_trains as the missing files (see attachment).

The title sequence will also fail to load for these parks, citing go_karts and corkscrew_trains as objects not found in console output.

The cause for this is likely the objects not being added to /usr/share/openrct2/object/rct1/ride on Linux installations in recent updates.

### Steps to reproduce

1. Attempt to load any park that contains RCT1 go karts or corkscrew trains.
2. The park will fail to load with an "Error loading objects for park" error message (see attachment).

### Attachments

Screenshot example for trying to load RCT1's Dynamite Dunes.
![Screenshot_20230217_195759](https://user-images.githubusercontent.com/7056104/219830563-fc705644-6e42-4ac2-9a6c-ffaa72875cf3.png)

Relevant screenshot of "/usr/share/openrct2/object/rct1/ride"
Notice that the rct1.ride "go_karts" and "corkscrew_trains" park objects are not present.
![Screenshot_20230217_201255](https://user-images.githubusercontent.com/7056104/219830661-d6e78dbc-e458-44d5-bb90-09970181a276.png)

**Comment by Gymnasiast** [comment]
Did you install OpenRCT2 via the PPA?

**Comment by MrCantaloupe** [comment]
> Did you install OpenRCT2 via the PPA?

Yes, the nightly PPA build.

It might be worth noting that this was also a problem before, four or five months ago, when the old RCT1 objects were added back into OpenRCT2.  It took a month or two for the files in /usr/share/ to be correctly updated.
https://github.com/OpenRCT2/OpenRCT2/issues/18461

Is there perhaps a problem with the build not putting objects in the right directory on Linux/Ubuntu?  I'm aware that the data folder is in a different location in Windows, with the rest of configuration data.

**Comment by Gymnasiast** [comment]
The problem is specific to PPA builds. Launchpad does not allow downloading files during the build process, so we have had to improvise by creating a separate repository with the objects. And we sometimes forget to update that one.

I have now pushed v1.3.8 of the objects and it should arrive in the next successful build. However, there also seem to be some build failures on Launchpad at the moment, so I cannot tell how soon that build will come.
